### DISCUSSION TITLE
How to combine the RL toolbox in Matlab with the S-function of Openfast?

### DISCUSSION THREAD

**Discussion by Domotopeng** [OpenFAST/openfast#884] [Q&A]
I want to use RL toolbox of the MATLAB to control Openfast, but I don't know how to combine the two to run in simulink?

**Comment by jjonkman** [comment]
Dear @Domotopeng,

I'm not really familiar with the Reinforcement Learning (RL) Toolbox of MATLAB, but from MathWorks' brief overview (https://www.mathworks.com/products/reinforcement-learning.html), it looks like the RL Toolbox itself is compatible with Simulink.  Is there an issue that you are aware of when one of the Simulink blocks is an OpenFAST S-Function?  If so, what is the issue?  If not, is your question better suited for MathWorks?

Best regards,
